src/components/ui/Chat/DetailedChannelList/DetailedChannelList.mdx

```diff
@@ -0,0 +1,53 @@
+import { Preview, Props } from '@storybook/addon-docs/blocks';
+import { ThemeProvider } from 'styled-components';
+
+import DetailedChannelList from './';
+import { _DetailedChannelList } from './DetailedChannelList.stories.tsx';
+import { lightTheme } from '../../../../theme/';
+import { GlobalStyles } from '../../../../theme/GlobalStyles';
+import PopOverItem from '../../PopOver/PopOverItem';
+
+# DetailedChannelList
+
+The DetailedChannelList component displays a column of chat channels along with last message's content and timestamp.
```

Comment:
Can you increase the documentation on the `DetailedChannelList` component? We already have a Channel List component and good documentation is important for builders to know the difference between the two. Please add more details like you added in the changelog.

> Add a new chat UI component DetailedChannelList which displays a list of channels with channel name, last message content and  timestamp, as well as unread channel badge.

Reply:
I guess my main concern is, is there any way we can make a more generic component to combine the two - there is a lot of code and documentation here for very similar components. Imagine the maintenance of both components compared to just one. Is it possible to add optional props to the Channel List component to make it bcomaptible?

Reply:
Thanks for the feedback. Discussed offline and made the change as suggested to update the existing ChannelList component instead of creating a new one.